rolepermissions/decorators.py

```diff
@@ -5,20 +5,24 @@
 from django.conf import settings
 from django.contrib.auth.views import redirect_to_login as dj_redirect_to_login
 from django.core.exceptions import PermissionDenied
+from django.shortcuts import redirect as dj_redirect
 
 from rolepermissions.checkers import has_role, has_permission
 from rolepermissions.utils import user_is_authenticated
 
 
-def has_role_decorator(role, redirect_to_login=None):
+def _role_permission_checker(function, subject, redirect_to_login, redirect_url):
```

Comment:
The two decorators were exactly the same, the only change was the used function to check so I made this to be just one function